browser/scripts/application.js

```diff
@@ -1082,6 +1082,9 @@ Application.prototype.onWindowResize = function() {
 		E2.app.player.core.renderer.update_viewport();
 	}
 
+	// Update preset list height so it scrolls correctly
+	$('.preset-list-container').height($('#left-nav').height() - $('#left-nav .nav-tabs').outerHeight(true) - $('#left-nav .tab-content .searchbox').outerHeight(true));
```

Comment:
please split this into multiple lines

`Strings longer than 80 characters should be written across multiple lines using string concatenation.`
https://github.com/airbnb/javascript/tree/master/es5#strings